soiree/app/controllers/group/playlists_controller.rb

```diff
@@ -29,6 +29,21 @@ def delete_group_redirect(playlist)
     redirect_to dashboard_path
   end
 
+  def delete_group_invites(playlist)
+    #When a group member deletes their portion of the group playlist, their image should be taken off other group
```

Comment:
I think this should happen automatically as the images are rendered dynamically by referencing the current group? If not let me know, ill revisit the problem